actions/commands.go

```diff
@@ -35,33 +35,20 @@ func Commands() {
 	app.Commands = []cli.Command{
 
 		{
-			Name:    "clone",
-			Aliases: []string{"c"},
-			Usage:   "Clone a template from github",
+			Name:  "project",
+			Usage: "Project management",
```

Comment:
Something more like `Manage Codewind projects` for now? Know this will probably change as we develop this!